packages/plugin-node-device/test/device.test.js

```diff
@@ -15,14 +15,16 @@ const VALID_NOTIFIER = { name: 't', version: '0', url: 'http://' }
 const ISO_8601 = /^\d{4}(-\d\d(-\d\d(T\d\d:\d\d(:\d\d)?(\.\d+)?(([+-]\d\d:\d\d)|Z)?)?)?)?$/i
 
 describe('plugin: node device', () => {
-  it('should set device = { hostname } and add a beforeSend callback which adds device time', done => {
+  it('should set device = { hostname }, capture the Node version and add a beforeSend callback which adds device time', done => {
```

Comment:
this test description could do with a bit of a tweak as the node version is part of the object that gets set as device

something like `'should set device = { hostname, runtimeVersions } and add a beforeSend…'`